### DISCUSSION TITLE
Welcome to detoxify Discussions!

### DISCUSSION THREAD

**Discussion by laurahanu** [unitaryai/detoxify#28] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SallyBean** [comment]
Great repo!

I have a question, I hope someone can help?

Are the toxicity scores provided by the Unitary models probability scores, in the same way that [perspective API](https://developers.perspectiveapi.com/s/about-the-api-score) returns these values?  

"The only score type currently offered is a probability score. It indicates how likely it is that a reader would perceive the comment provided in the request as containing the given attribute.  For each attribute, the scores provided represent a probability, with a value between 0 and 1. A higher score indicates a greater likelihood that a reader would perceive the comment as containing the given attribute. For example, a comment like “You are an idiot” may receive a probability score of 0.8 for attribute TOXICITY, indicating that 8 out of 10 people would perceive that comment as toxic. "

Or do they represent the extent of the toxicity?

Thanks so much!

**Comment by Rohan101314** [comment]
I could not get detoxify to work on lamda and had a question.

{"errorMessage": "/var/task/nvidia/cufft/lib/libcufft.so.10: failed to map segment from shared object", "errorType": "OSError", "requestId": 

The error message I get is posted above and I am not sure how to fix this. I am trying to import detoxify and deploy it via a container on aws. The library has gpu bound assets and causes this issue according to me. How do I workaround this on lambda?

I have tried changing models but, the imports stick, causing the issue. I was thinking about either using sagemaker or trying to get anaconda to work around by installing nvidea based packages. 

I have been stuck for a while on this and am unsure about how I could get this to work on lambda without expanding to other technologies?
